wurf_android_copy_lib_to_app.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""Tool for copying jni libraries to the appropriate Android app folder."""
+
+import os
+import shutil
+from waflib.TaskGen import feature
+from waflib.TaskGen import after_method
+
+from waflib import Task, Errors, Logs
+
+
+@feature('android_copy_lib_to_app')
+@after_method('apply_link')
+def android_copy_lib_to_app(self):
+    """
+    Copy library to Android application.
+
+    When building an Android application from the IDE native libraries placed
+    in the libs folder will be packaged in the apk. So we copy the shared libs
+    we build there.
+    """
+    if not self.bld.is_mkspec_platform('android'):
+        return
+
+    input_librarys = self.link_task.outputs
+    output_librarys = []
+    for input_library in input_librarys:
+        project_path = self.path.parent.abspath()
+        output_library = os.path.basename(input_library.abspath())
+        output_library = self.bld.root.make_node(
+            os.path.abspath(os.path.join(
+                project_path,
+                'app',
```

Comment:
I think we can set any parameter when we define the task:
```
bld(features='cxx cxxshlib android_copy_lib_to_app',
    source='native_media_receiver.cpp',
    target='native_media_receiver',
    export_includes=['jni'],
    install_path=None,
    copy_path='../app/src/main/jniLibs/armeabi',
    lib=['android'],
    use=['wurf_it'])
```
We also need to figure out if we want a relative path from the wscript_build, or from the project root.